infra/logger.go

```diff
@@ -0,0 +1,21 @@
+package infra
```

Comment:
it's really not necessary to extract this into a function - it's simple, and only used once in `main`